src/features/customerList/CustomerListContainer.tsx

```diff
@@ -87,6 +88,18 @@ const transformSearchBy = (searchBy: SearchBy, value: string): SearchByTransform
       invoicingTypes: [value],
     };
   }
+  if (searchBy === SearchBy.STICKER_NUMBER_SEASON) {
+    if (value.indexOf(' ') === -1) {
```

Comment:
trivial: I know this is the old and common way to check the (non-)presense, but I would still use `includes` instead, because it says it more clear what it does. Most probably it does the very same thing, since both are pure vanilla JS.

```suggestion
    if (!value.includes(' ')) {
```